extensions/sns/Cargo.toml

```diff
@@ -6,14 +6,14 @@ edition.workspace = true
 license.workspace = true
 repository.workspace = true
 publish.workspace = true
-build = "build.rs"
 
 [build-dependencies]
 dfx-extensions-utils.workspace = true
 
 [dependencies]
 dfx-core.workspace = true
 dfx-extensions-utils.workspace = true
+ic-sns-cli.workspace = true
```

Comment:
Probably need to remove "sns-cli" from the `[package.metadata.dist]` section at the end of this file

Reply:
Done!